keras_cv/models/object_detection/__test_utils__.py

```diff
@@ -21,13 +21,21 @@ def _create_bounding_box_dataset(
 ):
     # Just about the easiest dataset you can have, all classes are 0, all boxes are
     # exactly the same.  [1, 1, 2, 2] are the coordinates in xyxy
-    xs = tf.ones((5, 256, 256, 3), dtype=tf.float32)
-    y_classes = tf.zeros((5, 10), dtype=tf.float32)
+    xs = tf.random.normal(shape=(1, 512, 512, 3), dtype=tf.float32)
+    xs = tf.tile(xs, [5, 1, 1, 1])
```

Comment:
Why the switch to make all 5 images identical?

Reply:
otherwise it is too hard to make the test consistently overfit the dataset and thus receive a good Recall score.  When you're fitting noise I believe you end up with too many competing signals.

This was before the batchnorm though, so maybe I revert and try again?

Reply:
Actually, this was already making 5 identical images, but before they were all ones and now they are random uniforms.  

This allows the model to actually learn features.  When convolving across an image of ones yields the sum of ones always for all feature spaces --- it doesn't allow the model to distinguish between what is a box and what is not a box.  Its actually an impossible distribution for a convolutional neural network to learn.  The old dataset was meant to be trivial but it is actually literally impossible to solve without calling all pixels boxes.

Similarly, when all of the ys are stacked on the same location it becomes impossible for the model to say "there are 9 objects on the exact same pixel". Basically these changes are necessary to test Recall, and also make the test set closer to a real OD dataset by allowing the model to learn features, and making the boxes actually have distinct locations.